templates/repo/issue/comment_tab.tmpl

```diff
@@ -1,17 +1,35 @@
-<div class="ui top tabular menu" data-write="write" data-preview="preview">
-	<a class="active item" data-tab="write">{{.locale.Tr "write"}}</a>
-	<a class="item" data-tab="preview" data-url="{{.Repository.HTMLURL}}/markdown" data-context="{{.RepoLink}}">{{.locale.Tr "preview"}}</a>
-</div>
-<div class="field">
-	<div class="ui bottom active tab" data-tab="write">
+{{if .Fields}}
+	<input type="hidden" name="template-file" value="{{.TemplateFile}}">
+	{{range .Fields}}
+		{{if eq .Type "input"}}
+			{{template "repo/issue/fields/input" .}}
+		{{else if eq .Type "markdown"}}
+			{{template "repo/issue/fields/markdown" .}}
+		{{else if eq .Type "textarea"}}
+			{{template "repo/issue/fields/textarea" .}}
+		{{else if eq .Type "dropdown"}}
+			{{template "repo/issue/fields/dropdown" .}}
+		{{else if eq .Type "checkboxes"}}
+			{{template "repo/issue/fields/checkboxes" .}}
+		{{end}}
+	{{end}}
+	<div class="field"></div> {{/* the upload dropzone expects a field above and closes to it, see dropzone.less:9 */}}
```

Comment:
It seems quite fragile

Reply:
I add a new class `.not-under-easymde` with the style `margin-top: 0`, it works, but I‘m not sure if I did it in the right way.